portfolio/portfolio.iml

```diff
@@ -0,0 +1,75 @@
+    <orderEntry type="library" name="Maven: com.google.cloud:google-cloud-core-http:1.70.0" level="project" />
+    <orderEntry type="library" name="Maven: com.google.auth:google-auth-library-credentials:0.15.0" level="project" />
+    <orderEntry type="library" name="Maven: com.google.auth:google-auth-library-oauth2-http:0.15.0" level="project" />
+    <orderEntry type="library" name="Maven: com.google.http-client:google-http-client-jackson2:1.29.0" level="project" />
+    <orderEntry type="library" name="Maven: com.fasterxml.jackson.core:jackson-core:2.9.6" level="project" />
+    <orderEntry type="library" name="Maven: com.google.api-client:google-api-client:1.27.0" level="project" />
+    <orderEntry type="library" name="Maven: com.google.oauth-client:google-oauth-client:1.27.0" level="project" />
+    <orderEntry type="library" name="Maven: com.google.http-client:google-http-client-appengine:1.29.0" level="project" />
+    <orderEntry type="library" name="Maven: com.google.api:gax-httpjson:0.61.0" level="project" />
+    <orderEntry type="library" name="Maven: io.opencensus:opencensus-api:0.19.2" level="project" />
+    <orderEntry type="library" name="Maven: io.opencensus:opencensus-contrib-http-util:0.19.2" level="project" />
+    <orderEntry type="library" name="Maven: com.google.cloud:google-cloud-core-grpc:1.70.0" level="project" />
+    <orderEntry type="library" name="Maven: io.grpc:grpc-protobuf:1.19.0" level="project" />
+    <orderEntry type="library" name="Maven: io.grpc:grpc-protobuf-lite:1.19.0" level="project" />
+    <orderEntry type="library" name="Maven: io.grpc:grpc-context:1.19.0" level="project" />
+    <orderEntry type="library" name="Maven: com.google.api:gax-grpc:1.44.0" level="project" />
+    <orderEntry type="library" name="Maven: io.grpc:grpc-alts:1.19.0" level="project" />
+    <orderEntry type="library" name="Maven: io.grpc:grpc-grpclb:1.19.0" level="project" />
+    <orderEntry type="library" name="Maven: org.apache.commons:commons-lang3:3.5" level="project" />
+    <orderEntry type="library" name="Maven: com.google.apis:google-api-services-translate:v2-rev20170525-1.27.0" level="project" />
+    <orderEntry type="library" name="Maven: com.google.api.grpc:proto-google-cloud-translate-v3beta1:0.53.0" level="project" />
+    <orderEntry type="library" name="Maven: io.grpc:grpc-netty-shaded:1.19.0" level="project" />
+    <orderEntry type="library" name="Maven: io.grpc:grpc-core:1.19.0" level="project" />
+    <orderEntry type="library" name="Maven: io.opencensus:opencensus-contrib-grpc-metrics:0.19.2" level="project" />
+    <orderEntry type="library" name="Maven: io.grpc:grpc-stub:1.19.0" level="project" />
+    <orderEntry type="library" name="Maven: io.grpc:grpc-auth:1.19.0" level="project" />
+    <orderEntry type="library" name="Maven: javax.annotation:javax.annotation-api:1.3.2" level="project" />
+    <orderEntry type="library" name="jquery-3.5.1" level="application" />
```

Comment:
are all of these needed?

Reply:
They were automatically added for some reason ?